<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TTTmainWindow</class>
 <widget class="QMainWindow" name="TTTmainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>461</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Viner Hand ITC</family>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="menu">
     <widget class="QPushButton" name="Start">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>190</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Viner Hand ITC</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Ustaw">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>270</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Viner Hand ITC</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ustaw</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Zakoncz">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>350</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Viner Hand ITC</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Zakończ</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>471</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Viner Hand ITC</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>kółko i krzyżyk</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="game">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>301</width>
        <height>32</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Viner Hand ITC</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
     <widget class="QComboBox" name="Tryb">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>270</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Viner Hand ITC</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1 vs 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1 vs AI</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>361</width>
        <height>57</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Viner Hand ITC</family>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>ZACZYNAMY!</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>150</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Viner Hand ITC</family>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QComboBox" name="Znak">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>270</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Viner Hand ITC</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>kółko</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>krzyżyk</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>220</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Viner Hand ITC</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Znak</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>160</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Viner Hand ITC</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>na</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>220</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Viner Hand ITC</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tryb</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Viner Hand ITC</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Podaj wielkość planszy:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>150</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Viner Hand ITC</family>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
